sql/core/src/test/scala/org/apache/spark/sql/internal/SQLConfSuite.scala

```diff
@@ -319,6 +320,11 @@ class SQLConfSuite extends QueryTest with SharedSparkSession {
     assert(e2.message.contains("Cannot modify the value of a static config"))
   }
 
+  test("SPARK-36643: Cannot modify the value of a Spark config") {
```

Comment:
Please fix the test title like: Show migration guide when attempting SparkConf entry in error message

Reply:
Yes done